packages/server/lib/project.coffee

```diff
@@ -0,0 +1,616 @@
+_           = require("lodash")
```

Comment:
changes to project.coffee should have its changes merged into project.js and then deleted